docs/CompatibleSoftware.rst

```diff
@@ -132,32 +310,6 @@ Edit->Assign Profile, and then select your new OCIO lut.
 See the the OCIO user guide `for details on baking ICC profiles for Photoshop
 <userguide-bakelut-photoshop>`__
```

Comment:
In `master` and `RB-1.1`, the file does not exist anymore!
So, the sentence could be removed.